web/share/update-languages.py

```diff
@@ -124,32 +118,31 @@ def update(self, po_files, lang2territory: str, threshold: int):
                 supported.append(locale)
 
         languages = [loc.language for loc in supported]
-        self.content["locales"] = dict()
         for locale in supported:
             include_territory = languages.count(locale.language) > 1
-            self.content["locales"][locale.code()] = locale.name(
-                include_territory)
+            self.content[locale.code()] = locale.name(include_territory)
 
     def write(self):
         with open(self.path, "w+") as file:
-            json.dump(self.content, file, indent=4, ensure_ascii=False)
+            json.dump(self.content, file, indent=4, ensure_ascii=False,
+                      sort_keys=True)
```

Comment:
The `sort_keys=True` option ensures the list is sorted and avoids random reordering of the items.